labs/src/androidMain/kotlin/com/dimension/maskbook/labs/data/RedPacketMethod.kt

```diff
@@ -45,25 +44,28 @@ class RedPacketMethod(
             .onEach { message ->
                 when (message.method) {
                     notifyRedPacket -> {
-                        val options = message.params?.decodeJson<RedPacketOptions>() ?: return@onEach
-
-                        val currentWallet = walletServices.currentWallet.firstOrNull()
-                            ?: return@onEach
-                        val currentChain = walletServices.currentChain.firstOrNull()
-                            ?: return@onEach
-
-                        val data = options.toRedPacketState(currentWallet, currentChain)
-                        if (data.canClaim) {
-                            Navigator.navigate(LabsRoute.RedPacket.LuckyDrop(options.encodeJson()))
-                        }
-
+                        // val options = message.params?.decodeJson<RedPacketOptions>() ?: return@onEach
+                        //
+                        // val currentWallet = walletServices.currentWallet.firstOrNull()
+                        //     ?: return@onEach
+                        // val currentChain = walletServices.currentChain.firstOrNull()
+                        //     ?: return@onEach
+                        //
+                        // val data = options.toRedPacketState(currentWallet, currentChain)
+                        // if (data.canClaim) {
+                        //     Navigator.navigate(LabsRoute.RedPacket.LuckyDrop(options.encodeJson(), requestRaw))
+                        // }
```

Comment:
if the code is keep for some reasons please add a comment with it , otherwise just delete it